rfcs/testharness_include_late_tests.md

```diff
@@ -0,0 +1,42 @@
+# RFC 141: Change testharness to include late tests
+
+## Summary
+
+Currently, "late tests" are
+[ignored](https://github.com/web-platform-tests/wpt/blob/c751476c04678faa79e598683f2a6e94d17cf9e0/resources/test/tests/unit/late-test.html#L10-L17).
+This RFC proposes that testharness should now include them.
+
+## Details
+
+Conside the following test:
+
+```
+\<script>
+window.onload = () => {
+  test((t) => { ... }, 'test 1');
+  test((t) => { ... }, 'test 2');
+  test((t) => { ... }, 'test 3');
+};
+\</script>
+```
+
+Currently, only test 1 will be run. The issue is that the testharness
+immediately adds a window load handler that marks all_loaded = true,
```

Comment:
Can you sprinkle some backticks around here to make the code formatted `like this`?